common/remote/sockfile.cpp

```diff
@@ -1756,7 +1756,18 @@ class CRemoteFile : public CRemoteBase, implements IFile
         initSendBuffer(sendBuffer);
         MemoryBuffer replyBuffer;
         sendBuffer.append((RemoteFileCommandType)RFCsetfileperms).append(filename).append(fPerms);
-        sendRemoteCommand(sendBuffer, replyBuffer);
+        try
+        {
+            sendRemoteCommand(sendBuffer, replyBuffer);
+        }
+        catch (IDAFS_Exception *e)
+        {
+            if (e->errorCode() == RFSERR_InvalidCommand)
+                WARNLOG("umask setFilePermissions (0%o) not supported on remote server", fPerms);
+            else
+                throw e;
```

Comment:
trivial: I don't think it makes any difference here, but it can in general.
I think it is better to rethrow the exception with throw; vs naming the caught type.